File: PROJECT/Models/DTOs/Internal/Orders/OrderDTO.cs
﻿using Models.DTOs.Internal.Products;

namespace Models.DTOs.Internal.Orders
{
    public class OrderDTO
    {
        public int Id { get; set; }
        public int StatusId { get; set; }
        public string ClientFirstName { get; set; }
        public string ClientLastName { get; set; }
        public bool IsPossibleDuplicate { get; set; }
        public string ClientAddress { get; set; }
        public List<ProductItemDTO> Products { get; set; }

    }
}

File: PROJECT/FRONTEND/Common/DTOs/OrderDTO.ts


export class OrderDTO { 
    constructor(obj: Partial<OrderDTO>) {
        Object.assign(this,obj);
    }

    
    public id: number;
    public statusId: number;
    public clientFirstName: string;
    public clientLastName: string;
    public isPossibleDuplicate: boolean;
    public clientAddress: string;
    public products: ProductItemDTO[];
}
